core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeExpressionBuilder.scala

```diff
@@ -309,6 +310,33 @@ class SnowflakeExpressionBuilder()
       frame_spec = frameSpec)
   }
 
+  // see: https://docs.snowflake.com/en/sql-reference/functions-analytic#list-of-window-functions
+  private val SnowflakeRankRelatedWindowFunctions = Set(
```

Comment:
```suggestion
  private val rankRelatedWindowFunctions = Set(
```
is it CONSTANT_CASE or a ClassCase? i'm confused what this is. let's remove `snowflake` prefix, as we're already in the snowflake package.